invenio_previewer/templates/semantic-ui/invenio_previewer/zip.html

```diff
@@ -11,78 +11,57 @@
 
 {%- block head %}
 {{super()}}
-<style>
-.panel-heading {
-  background-image: radial-gradient(circle farthest-corner at left top, #0B5FB1 0%, #0B5FB1 100%) !important;
-  border-top-right-radius: 0;
-  border-top-left-radius: 0;
-  padding-top: 5px;
-  padding-bottom: 5px;
-}
-.panel-heading-alert {
-  background-image: radial-gradient(circle farthest-corner at left top, #B20000 0%, #B20000 100%) !important;
-}
-.panel-title {
-  color: hsl(0,0%,85%);
-  font-size: 14px;
-}
-.panel-default {
-  border-color: #fff;
-  border: 0;
-  box-shadow: none;
-}
-#fullScreenMode:hover {
-  background-color: rgba(0,0,200,0.5);
-  border-radius: 3px;
-}
+
+<style type="text/css">
+  .message { 
+    box-sizing: content-box;
+  }
+
+  .right-align {
+    text-align: right;
```

Comment:
minor: maybe the semantic-ui class "left aligned" or "left floated" would work? Note that they might require some grid changes, for example: https://semantic-ui.com/collections/grid.html  in the "clearing content section"

```
<div class="ui grid">
  <div class="four column row">
    <div class="left floated column"></div>
    <div class="right floated column"></div>
  </div>
</div>
```

Reply:
I changed it in the header but not in the file list because it was complicated because of the file loop. If you think it is worth it I can change it there too.